### PR TITLE
Create FT 2.0 TCK runner project

### PR DESCRIPTION
**Pull request by Azquelt**
TCK only runs when experimental FAT tests are requested

### CODE REVIEW

**Comment by Azquelt** [comment]
#build

**Comment by Azquelt** [comment]
> Please code review feature-related files, @OpenLiberty/delivery-approvers

I'm not sure what feature-related files this is referring to. This PR does not deliver any shipped code.

**Comment by tevans78** [comment]
> > Please code review feature-related files, @OpenLiberty/delivery-approvers
> 
> I'm not sure what feature-related files this is referring to. This PR does not deliver any shipped code.

That's a sort of bug that Wendy hasn't gotten around to fixing yet. It is flagging up changes to bnd files as well as feature files ... so you can imaging how much extra email I am getting at the moment!

File: dev/com.ibm.ws.microprofile.faulttolerance.2.0_fat_tck/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#*******************************************************************************
+# Copyright (c) 2018 IBM Corporation and others.
```
Comment (tevans78) [inline dev/com.ibm.ws.microprofile.faulttolerance.2.0_fat_tck/bnd.bnd]: date

File: dev/com.ibm.ws.microprofile.faulttolerance.2.0_fat_tck/build.gradle [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
```
Comment (tevans78) [inline dev/com.ibm.ws.microprofile.faulttolerance.2.0_fat_tck/build.gradle]: date

File: dev/com.ibm.ws.microprofile.faulttolerance.2.0_fat_tck/fat/src/com/ibm/ws/microprofile/faulttolerance/tck/FaultToleranceTck20Launcher.java [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+                                        // org.jboss.weld.exceptions.DefinitionException
+    }
+
+    /**
+     * Run the TCK (controlled by autoFVT/publish/tckRunner/tcl/tck-suite.html)
+     *
+     * @throws Exception
+     */
+    @Test
+    @Mode(TestMode.EXPERIMENTAL)
```
Comment (tevans78) [inline dev/com.ibm.ws.microprofile.faulttolerance.2.0_fat_tck/fat/src/com/ibm/ws/microprofile/faulttolerance/tck/FaultToleranceTck20Launcher.java]: is that the mode you want?

Comment (Azquelt) [reply dev/com.ibm.ws.microprofile.faulttolerance.2.0_fat_tck/fat/src/com/ibm/ws/microprofile/faulttolerance/tck/FaultToleranceTck20Launcher.java]: Yes, for the moment.

File: dev/com.ibm.ws.microprofile.faulttolerance.2.0_fat_tck/build.gradle (hunk 2 above)
Comment (Azquelt) [reply dev/com.ibm.ws.microprofile.faulttolerance.2.0_fat_tck/build.gradle]: done

File: dev/com.ibm.ws.microprofile.faulttolerance.2.0_fat_tck/bnd.bnd (hunk 1 above)
Comment (Azquelt) [reply dev/com.ibm.ws.microprofile.faulttolerance.2.0_fat_tck/bnd.bnd]: done

**Comment by Azquelt** [comment]
#build

**Comment by Azquelt** [comment]
#build

**Comment by Azquelt** [comment]
Failures in the build do not appear to be related to this change.

**Pull request merged**